cookbooks/providers/application/providers/default.rb

```diff
@@ -1,5 +1,26 @@
 action :install do
 
+  if new_resource.letsencrypt
+    log %{Install certbot-auto...}
+    execute 'wget certbot-auto' do
+      user 'root'
+      command 'wget https://dl.eff.org/certbot-auto -O /root/certbot-auto && chmod a+x /root/certbot-auto && mv /root/certbot-auto /usr/local/bin/certbot-auto'
```

Comment:
I prefer to use native chef resources for this. the `chmod` is included in the `remote_file` cookbook as `mode`. And the `mv` is probably not needed because you should be able to download right into `/usr/local/bin/certbot-auto` instead.